extensions/vertx-http/deployment/pom.xml

```diff
@@ -86,6 +111,268 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            
+            <!-- So we don't have to keep a version of these javascript libs in our repo -->
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.6.1</version>
+                <executions>
+                    <!-- Font awesome -->
+                    <execution>
+                        <id>install-fontawesome-css</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://unpkg.com/@fortawesome/fontawesome-free@${fontawesome.version}/css/all.min.css</url>
```

Comment:
So I'm not a big fan of this. Keep in mind that some restricted environments have very strict rules on where you can download from (we even have reports of people allowed to download from Central but not the Gradle plugin portal).

So... since you add them as dependencies, why don't we use the dependency plugin instead with something similar to https://maven.apache.org/plugins/maven-dependency-plugin/examples/unpacking-artifacts.html ?

Reply:
The users should never have to download this. It get's downloaded when we build this module. So only people building Quarkus. But let me look into the dependency-plugin. I did do it like that long ago. Can not remember why I switched.

Reply:
Not sure the dependency plugin would help here, since we're talking absolute URLs and not GAVs

Reply:
@gastaldi I'm talking about unzipping the files from the webjars.

Reply:
Ah cool, yeah I agree